setup.cfg

```diff
@@ -20,22 +20,21 @@ license_files = LICENSE
 # TODO(xuanwn): include all files in src/python/grpcio/grpc
 [pytype]
 inputs =
-    src/python/grpcio/grpc/experimental
-    src/python/grpcio/grpc
-    src/python/grpcio_tests/tests_aio
-    src/python/grpcio_observability/grpc_observability
-    examples/python/auth
-    examples/python/helloworld
+    src/python/grpcio/grpc/[!abe]*/
 exclude =
     **/*_pb2.py
-    src/python/grpcio/grpc/framework
-    src/python/grpcio/grpc/aio
-    src/python/grpcio/grpc/beta
-    src/python/grpcio/grpc/__init__.py
-    src/python/grpcio/grpc/_simple_stubs.py
 
 # NOTE(lidiz)
 # import-error: C extension triggers import-error.
 # module-attr: pytype cannot understand the namespace packages by Google.
 # attribute-error: Data classes in grpc module doesn't specify attributes.
 disable = "import-error,module-attr,attribute-error"
+
+[mypy]
+cache_dir = /dev/null
+files =
+    src/python/grpcio/grpc/[!abe]*/
```

Comment:
Again, what's going on with this regex?  [It's not even clear to me that `mypy` supports regexes in its config](https://mypy.readthedocs.io/en/stable/config_file.html#confval-files). And even if it did, this shouldn't match anything.